packages/babel-plugin-transform-typescript/src/enum.js

```diff
@@ -88,21 +90,27 @@ function enumFill(path, t, id) {
  *     Z = X | Y,
  *   }
  */
-type PreviousEnumMembers = { [name: string]: number | typeof undefined };
+type PreviousEnumMembers = { [name: string]: number | string | undefined };
```

Comment:
Can this actually be`undefined`? Looking at the `translateEnumValues` it seems to me that values are only set when they are a number.